### ISSUE TITLE
The question about the architecture of PTV3-object and the minimal resource usage for updating PTV3-object parameters

### ISSUE THREAD

**Issue by Fly-Eyes** [Pointcept/SAMPart3D#24]
I'm very impressed with your research. I want to use PTV3-object as the backbone of  my work to extract features of 3D object and I want to finetune PTV3-object network, so I want to  know the minimal resource usage for updating PTV3-object parameters and the architectural and parametric differences between PTV3-object and PTV3. I read the paper and find that you take 7 days on eight A800 GPUs in the large-scale pre-training stage. I don't have such rich resource.  Are A800 GPUs strictly necessary for my requirements? I only have 24G 3090 or 48G L20.

**Comment by yhyang-myron** [comment]
Hi, Thanks for your interest in our work!

A800 GPUs are not strictly necessary. You may try to finetune it on 3090 or L20.

**Comment by Fly-Eyes** [comment]
Thank you for your answers

**Comment by Fly-Eyes** [comment]
Hello, I've been pondering a new question these past few days regarding point cloud processing during the Large-scale Pre-training phase. Unlike the Sample-specific Fine-tuning phase where we input a single 3D model, batch-based processing of 3D models requires the same number of points across the batch. I'm curious whether you skipped the GridSample operation on point clouds during the Pre-training phase, since the number of points after downsampling would be unpredictable. Alternatively, did you implement padding and masking before feeding data into Ptv3-object? I'd like to understand how you approached batch training for Ptv3-object.
